cli/src/command/wallet.rs

```diff
@@ -238,14 +238,22 @@ pub async fn node_info_command(storage_path: &Path) -> Result<Wallet, Error> {
 pub async fn restore_command(storage_path: &Path, snapshot_path: &Path, backup_path: &Path) -> Result<Wallet, Error> {
     check_file_exists(backup_path).await?;
 
-    let password = get_password("Stronghold password", false)?;
-    let secret_manager = SecretManager::Stronghold(
-        StrongholdSecretManager::builder()
-            .password(password.clone())
-            .build(snapshot_path)?,
-    );
-    let wallet = Wallet::builder()
-        .with_secret_manager(secret_manager)
+    let mut builder = Wallet::builder();
+    if check_file_exists(snapshot_path).await.is_ok() {
+        println_log_info!(
+            "Detected a stronghold file at {}, requiring unlock..",
```

Comment:
The messages here are a little weird. Maybe go with
```
Detected a stronghold file at {}. Enter password to unlock:
```

and not sure about "Detected Stronghold password" below

Reply:
The"Detected Stronghold password" is for the prompt to unlock the current stronghold (named detected)
Afterwards, you get a prompt as well to unlock the stronghold you import ("imported stronghold)
The detected and imported name is important to not cause confusions when both prompts trigger

Reply:
Sure, but the phrase is confusing. It reads like a statement rather than a prompt

Reply:
Updated